src/features/projectsV2/ProjectSnippet/microComponents/ProjectBadge.tsx

```diff
@@ -111,11 +117,16 @@ const ProjectBadge = ({ isApproved, isTopProject, allowDonations }: Props) => {
   if (!badgeConfigurations) return null;
 
   const { icon, title, displayPopup, badgeType } = badgeConfigurations;
+
+  const shouldShowPopup = useCallback(
+    () => showPopup && displayPopup,
+    [showPopup, displayPopup]
+  );
```

Comment:
Remove the `useCallback`. This is a simple calculation, and is unlikely to cause a performance issue even with 300+ components being rendered at a time.

Reply:
I updated this to a simple conditional statement instead of a function.